models/twilio__account_overview.sql

```diff
@@ -44,13 +47,18 @@ select
     {% endfor %}
     
     count(messages.message_id) as total_messages,
-    round( cast(sum(messages.price) as {{ dbt.type_numeric ()}}), 2) * -1 as total_messages_spend,
-    round( cast(sum(usage_record.price) as {{ dbt.type_numeric ()}}), 2) as total_account_spend
+    round( cast(sum(messages.price) as {{ dbt.type_numeric() }}), 2) * -1 as total_messages_spend
+    
+    {% if var('using_twilio_usage_record', True) %}
+    , round( cast(sum(usage_record.price) as {{ dbt.type_numeric() }}), 2) as total_account_spend
```

Comment:
Can you update the documentation for this field [here](https://github.com/fivetran/dbt_twilio/blob/71baaccc6f4e668e6cfed9dbe49b97df917b7f5e/models/twilio.yml#L167-L168) to add a disclaimer that this will only be present if the `using_twilio_usage_record` variable is defined as true (default being true).

Reply:
added